src/OrchardCore.Modules/OrchardCore.Media/Indexing/PdfMediaFileTextProvider.cs

```diff
@@ -0,0 +1,32 @@
+using System.IO;
+using Cysharp.Text;
+using UglyToad.PdfPig;
+
+namespace OrchardCore.Media.Indexing
+{
+    public class PdfMediaFileTextProvider : IMediaFileTextProvider
+    {
+        public bool CanHandle(string path)
+        {
+            return path.EndsWith(".pdf", System.StringComparison.OrdinalIgnoreCase);
```

Comment:
```csharp
private static readonly string PdfExtension = ".PDF";

public bool CanHandle(string path) => path.EndsWith(PdfExtension, System.StringComparison.OrdinalIgnoreCase);
```

Reply:
Why would it be useful to pull out a single-use string into a private field?

As for having an expression-bodied method, I actually prefer that personally but I think the practice in Orchard is not to do it (unlike for properties).

Reply:
This one I won't debate as it is mostly a preference than anything. No value added.